app/src/main/java/com/deveddy/clujbike/data/repository/specifications/RealmSpecification.java

```diff
@@ -0,0 +1,11 @@
+package com.deveddy.clujbike.data.repository.specifications;
+
+import com.deveddy.clujbike.data.repository.models.StationRealm;
+
+import io.realm.Realm;
+import io.realm.RealmQuery;
+
+public interface RealmSpecification extends Specification {
```

Comment:
Maybe would be worth it generifying this interface so it can be used in the future for other types as well, not only `StationRealm`